tests/Providers/Trustpilot/ServiceActionsTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Providers\Trustpilot;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Providers\Trustpilot\ServiceActions;
+use PodPoint\Reviews\Tests\TestCase;
+
+class ServiceActionsTest extends TestCase
+{
+    /**
+     * Constructor should be instance of APiClientInterface and it should assign to apiClient property.
+     */
+    public function testConstruct()
+    {
+        $action = new Mocked_ServiceActions($this->getMockedApiClient());
+
+        $this->assertInstanceOf(ActionsInterface::class, $action);
+        $this->assertInstanceOf(ApiClientInterface::class, $action->getApiClient());
+    }
+
+    /**
+     * Making sure business id is setters and getters setting property as expected.
+     */
+    public function testBusinessUnitId()
+    {
+        $action = new ServiceActions($this->getMockedApiClient());
+
+        $businessId = 'foo-bar-123';
+        $action->setBusinessUnitId($businessId);
+
+        $this->assertEquals('foo-bar-123', $action->getBusinessUnitId());
+    }
+
+    /**
+     * Making sure the invite method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testInvite()
+    {
+        $options = [
+            'referenceNumber' => 'reference-number-321',
+            'consumerEmail' => 'customer@email.com',
+            'consumerName' => 'Customer Name',
+        ];
+
+        $response = $this->getMockedResponse('{"status": "200", "message": "accepted"}');
+        $apiClient = $this->getMockedApiClient();
+        $apiClient->shouldReceive('sendRequest')->withAnyArgs()->andReturn($response);
+
+        $action = new ServiceActions($apiClient);
+        $action->setBusinessUnitId('foo-bar-321');
+
+        $inviteResponse = $action->invite($options);
+
+        $expectedResult = [
+            "status" => "200",
+            "message" => "accepted",
+        ];
+
+        $this->assertEquals($expectedResult, $inviteResponse);
+    }
+
+    /**
+     * Making sure the getReviews method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testGetReviews()
+    {
+        $options = [];
```

Comment:
Same point here: "Can we add a test here where we check that there is no error (and that we get all reviews as expected) when we don't pass an argument to the getReviews() function?"

I think this would be best solved if you pass options to the test with a dataprovider that passes different things like `[]` `null` `[some actual stuff]`